client/src/App.tsx

```diff
@@ -27,6 +28,10 @@ function App(): JSX.Element {
                 <ProtectedRoute exact path="/dashboard">
                   <Dashboard />
                 </ProtectedRoute>
+                <Route exact path="/edit-profile">
+                  <EditProfile />
+                </Route>
+                {/* <Route exact path="/profile" component={ProfileSetting} /> */}
```

Comment:
In the future, please make sure you uncomment or remove this accordingly

Reply:
Fixed